docs/docs/releases/0.1.6.md

```diff
@@ -0,0 +1,36 @@
+# Mathesar 0.1.6
+
+## Summary
+
+Mathesar 0.1.6 brings compatibility with Python versions 3.10 and 3.11, PostgreSQL 16 support, ease of use while working with longer texts on the record page, as well as Japanese translations within the application.
+
+_This page provides a comprehensive list of all changes in the release._
+
+## Upgrading to 0.1.6
+
+See our guide on [upgrading Mathesar to 0.1.6](../administration/upgrade/0.1.6.md).
+
+## Improvements
+
+- Mathesar is now compatible with Python versions: 3.10 and 3.11 along with 3.9 _[#3478](https://github.com/mathesar-foundation/mathesar/pull/3478)_
+- Added support for Japanese translations _[#3486](https://github.com/mathesar-foundation/mathesar/pull/3486)_, _[#3484](https://github.com/mathesar-foundation/mathesar/pull/3484)_, _[#3483](https://github.com/mathesar-foundation/mathesar/pull/3483)_, _[#3472](https://github.com/mathesar-foundation/mathesar/pull/3472)_
+- NodeJS is no longer a requirement for building Mathesar from source _[#3474](https://github.com/mathesar-foundation/mathesar/pull/3474)_
+- Text fields now auto-expands on the record page to accommodate longer texts _[#3470](https://github.com/mathesar-foundation/mathesar/pull/3470)_
+
+## Bug fixes
+
+- Fixed regression within the record selector where records won’t get filtered based on input _[#3488](https://github.com/mathesar-foundation/mathesar/pull/3488)_
+- Fixed failures while adding a connection along with sample schema(s), whose name(s) would collide with schema(s) already present in the database _[#3490](https://github.com/mathesar-foundation/mathesar/pull/3490)_
+- Fixed record selector's text inputs from growing taller _[#3495](https://github.com/mathesar-foundation/mathesar/pull/3495)_
+- Fixed CSRF failures when the app is behind a reverse proxy _[#3499](https://github.com/mathesar-foundation/mathesar/pull/3499)_
+- Fixed layout issues while using Japanese translations _[#3501](https://github.com/mathesar-foundation/mathesar/pull/3501)_
+
+## Maintenance
+
+- Added a health check endpoint to Mathesar _[#3479](https://github.com/mathesar-foundation/mathesar/pull/3479)_
+- Added PG 16 to testing matrix _[#3480](https://github.com/mathesar-foundation/mathesar/pull/3480)_
+- Added MkDocs edit URI _[#3482](https://github.com/mathesar-foundation/mathesar/pull/3482)_
+- Bumped Django from 4.2.8 to 4.2.10 _[#3492](https://github.com/mathesar-foundation/mathesar/pull/3492)_
+- Removed stale code coverage badge _[#3491](https://github.com/mathesar-foundation/mathesar/pull/3491)_
+- Added GitHub workflow to attach pre-built static files with new releases _[#3489](https://github.com/mathesar-foundation/mathesar/pull/3489)_ 
```

Comment:
This is actually the part (along with docs) that removes the NodeJS dependency for building from source.